admin/modules/user/groups.php

```diff
@@ -743,7 +743,15 @@
 			// Log admin action
 			log_admin_action($gid, htmlspecialchars_uni($mybb->input['title']));
 
-			flash_message($lang->success_group_created, 'success');
+			$groups = $cache->read('usergroups');
+			$grouptitles = array_column($groups, 'title');
+
+			$message = $lang->success_group_created;
+			if(in_array($mybb->input['title'], $grouptitles) && count(array_keys($grouptitles, $mybb->input['title'])) > 1) {
```

Comment:
The bracket should be in a new line

Reply:
Fixed in 85e640af00f8be38b5ac5e50fcfea3d7850c5727